### ISSUE TITLE
Picture or avatar as alternative for non camera users

### ISSUE THREAD

**Issue by bra1n** [ThePandemoniumInstitute/botc-release#38]
Sure it should be optional for people to use live camera on a game on the internet with potentially many strangers, but I think people should at least be able to use a picture or a selectable character avatar (maybe with simular art style as the base game) as a alternative to webcam feed.

It would be a great way to differenciate people, especially before we tag each player we talk to with tokens. Sometimes its hard to remember people with only a blank cirkle on top of they're name, especially in large games.

(There are even programs that animates a character with your live webcam feed, but I guess that would be to hard to implement to the app in the near future).

**Comment by VinylVault** [comment]
this should also be used if someone has their camera turned off, rather than seeing the `camera off` icon

**Comment by bra1n** [comment]
Yeah, maybe. It would need an indicator still to show that the user has their camera only "muted" rather than disabled.

**Comment by bra1n** [comment]
Done!
